recipes/cromwell/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "40" %}
+{% set sha256_cromwell = "8e1f9c8777bb3b860fd1bf6b6dc05a51145889620f0510cb2fa89aa7e1dfd3ec" %}
+{% set sha256_womtool = "1904ebc87c605b4330ddd88770b522104d0bd506be2b9bcfe8424e90d38e4fb9" %}
+
+package:
+  name: cromwell
+  version: 0.{{ version }}
+
+source:
+  - url: https://github.com/broadinstitute/cromwell/releases/download/{{ version }}/cromwell-{{ version }}.jar
+    sha256: {{ sha256_cromwell }}
```

If you move the hash here and delete the variable I think the bot will stand a better chance of automatically updating:
```suggestion
    sha256: 8e1f9c8777bb3b860fd1bf6b6dc05a51145889620f0510cb2fa89aa7e1dfd3ec
```